src/common/format.ts

```diff
@@ -110,6 +127,20 @@ export interface RtCallSite {
     r: RtType;
 }
 
+export interface RtAliasType {
+    TΦ: typeof T_ALIAS;
+    name: string;
+    a: AliasToken;
```

Comment:
I understand why you _want_ the optional forward referencing -- its because you are extending the Constructor/InterfaceToken automated import/export system to include AliasTokens and those would be affected by circular import loops. So why not just have `a: () => AliasToken` here?

Reply:
Same as above,
will change everything to
```
()=>RtType
```